### PR TITLE
Update/formattime

### PR DESCRIPTION
**Pull request by capetillo**
Updates utils functions `formatDate`, `formatTime`, and `formatDateMMDDYY`  into one:`formatDateTime` to further simplify the utility functionality. 
Updates tests too

### VISUALS
**Date and time rendering as it should**
Screenshot 2024-12-10 at 12 47 44 PM

### CODE REVIEW

**Review by LTDakin — APPROVED** [review]

File: src/utils/formatTime.js [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
   const date = new Date(dateString)
-  options = { year: 'numeric', month: 'long', day: 'numeric' }
-  return date.toLocaleDateString('en-US', options)
-}
-
-function formatTime (timeString) {
-  const date = new Date(timeString)
-  const options = { hour: 'numeric', minute: 'numeric' }
-  return date.toLocaleTimeString('en-US', options)
+  if (options && (options.hour || options.minute)) {
```
[the remark this answers is not in the document]
Comment (capetillo) [reply src/utils/formatTime.js:4]: it was the same function 3 times that formatted time in 3 different way so now it just takes the 2 values as params: the date string and options (i.e. how we want to format the time)

Comment (mgdaily) [inline src/utils/formatTime.js:4]: can you comment this to say why we need this conditional statement, or what the correct format of `options` is? might help a future programmer. alternatively, you can expose the specific options keys as function parameters.

Comment (capetillo) [reply src/utils/formatTime.js:4]: yes I can do that!

Comment (capetillo) [reply src/utils/formatTime.js:4]: there are so many options that I think it would be excessive to pass them as keys hah but I'm adding the comment now

**Pull request merged**